modules/tinymce/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed an exception getting thrown when the number of `col` elements didn't match the number of columns in a table #TINY-7041 #TINY-8011
 - As of Mozilla Firefox 91, toggling fullscreen mode with `toolbar_sticky` enabled would cause the toolbar to disappear #TINY-7873
 - Inserting content into a `contenteditable="true"` element that was contained within a `contenteditable="false"` element would move the selection to an incorrect location #TINY-7842
+- The `wordcount` plugin no longer treats soft hyphens (`&shy;` entities) as a word break #TINY-7908
```

Comment:
Should this be in the `Changed` section instead of the `Fixed` section?

Reply:
I would say fixed because the old behaviour was a bug. Hyphens shouldn't have ever counted towards word breaks. Happy to change this if you feel strongly though

Reply:
I'm happy to leave it in fixed, but we'll need to reword this entry then, as this is very much so a `Changed` entry atm.

Reply:
```suggestion
- The `wordcount` plugin was incorrectly treating soft hyphens (`&shy;` entities) as word breaks #TINY-7908
```

Reply:
LGTM :+1: